pkg/kubelet/kubelet_node_status.go

```diff
@@ -518,11 +519,15 @@ func (kl *Kubelet) defaultNodeStatusFuncs() []func(*v1.Node) error {
 	if utilfeature.DefaultFeatureGate.Enabled(features.AttachVolumeLimit) {
 		setters = append(setters, nodestatus.VolumeLimits(kl.volumePluginMgr.ListVolumePluginWithLimits))
 	}
+	var currentServingCert func() *tls.Certificate
+	if kl.serverCertificateManager != nil {
```

Comment:
Just a doubt that how we are going to handle if k1.serverCertificateManager = nil.? Thanks in advance

Reply:
When `k1.serverCertificateManager == nil`, serving certificate rotation is not enabled.
Node status will not be tied to serving certificate status. So (assuming all other checks here pass) Node will come up `Ready`.